Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:trusty
+
+RUN apt-get -y update
+RUN apt-get -y upgrade
+
+#Use: --build-arg VEXPRESS_IMAGE=scripts/core-image-full-cmdline-vexpress-qemu.sdimg --build-arg UBOOT_ELF=u-boot.elf when building
```

Comment:
How come these files end up in scripts? Maybe provide a script for building docker image so that we don't need to set these variables explicitly in the command line.